pkgs/tools/networking/shadowsocks-libev/default.nix

```diff
@@ -8,11 +8,12 @@ stdenv.mkDerivation rec {
   version = "3.2.0";
 
   # Git tag includes CMake build files which are much more convenient.
-  # fetchgit because submodules.
-  src = fetchgit {
-    url = "https://github.com/shadowsocks/shadowsocks-libev";
+  src = fetchFromGitHub {
+    owner = "shadowsocks";
+    repo = "shadowsocks-libev";
     rev = "refs/tags/v${version}";
```

Comment:
If you point at `v${version}` you get the archive tarball of the repository which contains the CMake build files too.